### ISSUE TITLE
Impossible to change Album Artist for entire album at once

### ISSUE THREAD

**Issue by gdsa32** [enzo1982/freac#317]
**Describe the bug**
When I import an entire album to the track list, I see no way to easily change the Album Artist tag for that album in one single step.
If I go to Tags tab, select the album, then go to Details tab below, the Album Artist field is disabled. It seems logic to me to be able to edit that field and change the Album Artist tag for all tracks in that album in one single step. 

**To Reproduce**
1. Import an album to the track list, preferably one where Artist and Album Artist tags are different, like a compilation or soundtrack
2. Click on Tabs tab, then on Albums tab
3. Click on the album
4. Click on Details tab, below the album list
5. The Album Artist field is disabled, making it impossible to change the Album Artist field for all tracks at once

**Expected behavior**
In step 5 above, I expected the Album Artist field to be editable.

![image](https://user-images.githubusercontent.com/39100541/170298536-f9b63dc6-cd7f-4afa-aa76-c1c933bedb40.png)

![image](https://user-images.githubusercontent.com/39100541/170298793-7f95f676-9e41-4aca-a7d2-1fd00c0d4aac.png)

**System (please complete the following information)**
 - OS: Windows 10 x64 v21H2
 - CPU: Intel Core i7 10700
 - RAM: 64 GB

**Comment by ciphersimian** [comment]
It's on the Basic tab when you have an Album selected (Artist).  Album Artist is greyed out in this scenario because it's redundant.

**Comment by gdsa32** [comment]
> It's on the Basic tab when you have an Album selected (Artist). Album Artist is greyed out in this scenario because it's redundant.

I disagree. The Basic tab has only the Artist field. But I'm referring to scenarios where Artist and Abum Artist fields are different. Think of Soundtrack albums, for example... In such case Album Artist is something like 'Various Artists' and Artist is unique to each track in the album. What I expect is a way to change only the Album Artist field, for all tracks at once, without affecting the Artist field. 

This is important for me, because I'm using <Album Artist> as part of the mask for destination folder. If Album Artist is not consistently tagged, we may end up with one track per folder during conversion.
